domain/include/cstone/domain/domain.hpp

```diff
@@ -439,14 +439,19 @@ class Domain
                                          manifest.rangeStart(ri), manifest.rangeEnd(ri)));
             }
         }
+
+		#define _unused(x) ((void)(x))
+		_unused(start);
+		_unused(end);
     }
 
     //! @brief return true if all array sizes are equal to value
     template<class... Arrays>
     static bool sizesAllEqualTo(std::size_t value, Arrays&... arrays)
     {
         std::array<std::size_t, sizeof...(Arrays)> sizes{arrays.size()...};
-        return std::count(begin(sizes), end(sizes), value) == sizes.size();
+
+        return (size_t)std::count(begin(sizes), end(sizes), value) == sizes.size();
```

Comment:
prefer function-style cast over C-style cast:
```suggestion
        return size_t(std::count(begin(sizes), end(sizes), value)) == sizes.size();
```

Reply:
Yes, sure. I worked with c-style for 2-years ... I still have automatic habits :)